sphinx/cmd/quickstart.py

```diff
@@ -12,6 +12,7 @@
 from __future__ import absolute_import
 
 import argparse
+import collections
```

Comment:
Personally, I prefer to import `OrderedDict` only.

Reply:
I'm following OpenStack coding standards which insist on [module, not object, imports](https://docs.openstack.org/hacking/latest/user/hacking.html#imports) as it results in less imports. Can fix though